cypress.config.ts

```diff
@@ -50,7 +50,7 @@ module.exports = defineConfig({
   e2e: {
     baseUrl: 'http://localhost:5601',
     specPattern: 'cypress/integration/**/*.spec.{js,jsx,ts,tsx}',
-    testIsolation: false,
+    testIsolation: true,
```

Comment:
i was wondering why this was set to false. If tests pass then yeah this should be set to true.

But to give some background knowledge. In other environments we set this to false. This is because some of our tests require authentication and that does not persist if this is set to true